### PR TITLE
Test PR please ignore

### PR DESCRIPTION
**Pull request by jwilander2**
Testing some things

### CODE REVIEW

**Comment by jwilander** [comment]
@jwilander should not get this

**Comment by jwilander** [comment]
@jwilander

**Comment by jwilander** [comment]
@jwilander should not get this2

**Comment by jwilander2** [comment]
@jwilander should get this

File: README.md [hunk 1]
```diff
@@ -221,4 +221,4 @@ The server needs to be available at `http://localhost:8065`. This can be overrid
 
 Finally, set "EnableOpenServer", "EnableCustomEmoji", "EnableLinkPreviews", "EnableUserAccessTokens" and "EnableOAuthServiceProvider" to `true` and "EnableOnlyAdminIntegrations" to `false` in your config.json. If you don't have a config.json yet, create it by copying default.json.
 
-With that set up, you can run the tests against your server with `npm run test-no-mock`.
+With that set up, you can run the tests against your server with `npm run test-no-mock`..
```
Comment (jwilander) [inline README.md]: Some comment about something

File: README.md [hunk 2]
```diff
@@ -221,4 +221,4 @@ The server needs to be available at `http://localhost:8065`. This can be overrid
 
 Finally, set "EnableOpenServer", "EnableCustomEmoji", "EnableLinkPreviews", "EnableUserAccessTokens" and "EnableOAuthServiceProvider" to `true` and "EnableOnlyAdminIntegrations" to `false` in your config.json. If you don't have a config.json yet, create it by copying default.json.
```
Comment (jwilander) [inline README.md]: Add single comment

**Pull request merged**